lib/active_merchant/billing/gateways/omise.rb

```diff
@@ -194,8 +194,10 @@ def post_data(parameters)
 
       def https_request(method, endpoint, parameters=nil, options={})
         raw_response = response = nil
+        req_headers  = headers(options)
```

Comment:
:+1: I would just add it to the `headers` method and just set a default value if needed as @girasquid mentioned below